experimental/devices/mfrc522/commands/registers.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package commands
+
+// Command register constants
```

Comment:
Technically, it is just Register contants, only one of all of them is a comment register. :)